sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/translation/SqlOperators.java

```diff
@@ -177,26 +205,25 @@ private static SqlUserDefinedFunction createUdfOperator(
       Class<?> methodClass,
       String methodName,
       SqlReturnTypeInference returnTypeInference,
-      List<RelDataType> paramTypes) {
+      List<RelDataType> paramTypes,
+      String funGroup) {
```

Comment:
nit: It seems like `SqlAnalyzer.ZETASQL_FUNCTION_GROUP_NAME` is the only thing ever passed to `funGroup` here and below. Should it really be an argument?

Reply:
The full implementation uses the group name to distinguish built-in functions from user-defined functions. (Whether that's a good idea or not is up for debate.)